Plugins/Mono/Mono_Handlers.cpp

```diff
@@ -396,4 +400,20 @@ int32_t ClosureActionDoCommand(uint32_t oid, uint64_t eventId)
     return 0;
 }
 
+std::string ISO88959ToUTF8(const char *str)
+{
+    std::string utf8("");
+    utf8.reserve(2*strlen(str) + 1);
+
+    for (; *str; ++str) {
```

Comment:
Actually that's my preferred style too. Slipped in since the original function was copypaste, though I then discovered it was written totally wrong in respects to the UTF-8 spec and needed to redo it.